test/server.spec.js

```diff
@@ -40,6 +40,7 @@ describe('Subgenerator server of entando JHipster blueprint', () => {
           buildTool: 'maven',
```

Comment:
I think we should add some tests to check if the files contain the right dependency management. So tests for Entando but also JHipster

Reply:
Tests added